activiti-cloud-service-common/activiti-cloud-service-messaging-config/src/main/java/org/activiti/cloud/common/messaging/config/FunctionBindingConfiguration.java

```diff
@@ -188,4 +198,24 @@ public Object postProcessAfterInitialization(Object bean, String beanName) throw
     }
 
     public interface BindingResolver extends Function<String, String> {}
+
+    @Bean
+    @ConditionalOnClass(JacksonMapper.class)
+    @Primary
+    public JacksonMapper jacksonMapper(@Autowired(required = false) ObjectMapper objectMapper) {
+        //temporary workaround for https://github.com/spring-cloud/spring-cloud-function/issues/1159
+        ObjectMapper copiedMapper;
+        try {
+            copiedMapper = objectMapper.copy();
+            //logic from AlfrescoWebAutoConfiguration.configureObjectMapperForBigDecimal
+            copiedMapper.configOverride(BigDecimal.class).setFormat(JsonFormat.Value.forShape(JsonFormat.Shape.STRING));
```

Comment:
Shouldn't this be called also in the case where a new Object mapper is instantiated (catch exception block)?

Reply:
agree, moved